From c37d3915c820ecc8db765b2a67e307e42e400734 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Mon, 19 Aug 2019 21:31:12 +0900
Subject: [PATCH] add salsa-ci.yml by default

It can be used without any customise, so not add .ex postfix
---
 lib/debian/salsa-ci.yml | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 lib/debian/salsa-ci.yml

diff --git a/lib/debian/salsa-ci.yml b/lib/debian/salsa-ci.yml
new file mode 100644
index 0000000..33c3a64
--- /dev/null
+++ b/lib/debian/salsa-ci.yml
@@ -0,0 +1,4 @@
+---
+include:
+  - https://salsa.debian.org/salsa-ci-team/pipeline/raw/master/salsa-ci.yml
+  - https://salsa.debian.org/salsa-ci-team/pipeline/raw/master/pipeline-jobs.yml
-- 
2.23.0.rc1

